src/openloco/windows/IndustryWindow.cpp

```diff
@@ -83,8 +83,8 @@ namespace openloco::ui::windows::industry
         };
 
         static widget_t widgets[] = {
-            commonWidgets(223, 136, string_ids::title_town),
-            make_widget({ 3, 44 }, { 195, 79 }, widget_type::viewport, 1, 0xFFFFFFFE),
+            commonWidgets(223, 137, string_ids::title_town),
```

Comment:
town??

Reply:
both town and industry windows use string_id 1359 `title_town` which I haven't renamed

Reply:
Ah, alright then.